react-common/components/theming/ThemeCard.tsx

```diff
@@ -0,0 +1,32 @@
+
+import * as React from "react";
+import { Button } from "../controls/Button";
+import { ThemePreview } from "./ThemePreview";
+
+interface ThemeCardProps {
+    theme: pxt.ColorThemeInfo;
+    onClick?: (theme: pxt.ColorThemeInfo) => void;
+}
+
+export class ThemeCard extends React.Component<ThemeCardProps> {
+    render() {
+        const { onClick, theme } = this.props;
+
+        return (
+            <div key={theme.id} className="theme-picker-item">
+                <Button
+                    className="ui card link card-selected theme-card"
```

Comment:
can we remove the semantic ui references? we've been doing a good job of keeping semantic ui out of react-common